lib/recurly.js

```diff
@@ -244,7 +249,7 @@ export class Recurly extends Emitter {
       this.config.currency = options.currency;
     }
 
-    this.config.coBrands = options.coBrands || [];
+    this.config.coBrands = options.coBrands || this.config.coBrands || [];
```

Comment:
This is needed because `configure` method can be called multiple times, with incremental configuration options, so the existing should remain untouched.